xlml/utils/tpu.py

```diff
@@ -239,7 +239,12 @@ def wait_for_tpu_deletion(qualified_name: str):
     creds, _ = google.auth.default()
     client = tpu_api.TpuClient(credentials=creds)
 
-    qr = client.get_queued_resource(name=qualified_name)
+    try:
+      qr = client.get_queued_resource(name=qualified_name)
+    except google.api_core.exceptions.NotFound:
+      logging.info(f'{qualified_name} was unexpectedly deleted')
```

Comment:
Can we update to something like below? So we know one direction for debugging in the future.

```
f'{qualified_name} was deleted by clean up DAG or unexpectedly'
```

Reply:
Done